content/understanding-urbit/urbit-id.md

```diff
@@ -9,27 +9,27 @@ hide_previous_title = "true"
 
-Your Urbit ID is a short, four-syllable name like `~ravmel-ropdyl` that you own with an eight-syllable master passkey like `~palfun-foslup-fallyn-balfus`. This name and key lets you log into Urbit OS, and it’s used to encrypt packets you send over the Urbit network. Soon it will also be the master key for a keychain that allows holding and sending of Bitcoin and other cryptocurrencies. Your Urbit ID and passkey belong to you like any other cryptographic assets. No one can take them away from you — just make sure to store the key safely.
+Your Urbit ID is a short, four-syllable name like `~ravmel-ropdyl` that you own with an eight-syllable master passkey like `~palfun-foslup-fallyn-balfus`. This name and key lets you log into Urbit OS, and it’s used to encrypt packets you send over the Urbit network. Soon it will also be a master key that allows holding and sending of Bitcoin and other cryptocurrencies. Your Urbit ID and passkey belong to you like any other cryptographic assets. No one can take them away from you (just make sure to store the key safely).
 
-The Urbit ID registry is live and deployed to the Ethereum blockchain. Urbit ID isn’t specifically wedded to Ethereum – someday we’d like it to be hosted by Urbit OS itself. Also, Urbit ID is the only component of the stack using Ethereum – your Urbit OS node is hosted wherever you choose. Anyway, the Urbit ID registry keeps track of who owns what, which keys are associated with which names, and enforces the rules of address distribution. We’ll get into the details a bit more below.
+The Urbit ID registry is live and deployed to the Ethereum blockchain. Urbit ID isn’t specifically wedded to Ethereum – someday we’d like it to be hosted by Urbit OS itself. Also, Urbit ID is the only component of the stack using Ethereum – your Urbit OS node is hosted wherever you choose. The primary function of the Urbit ID registry is to keep track of who owns what, to specify which keys are associated with which names, and to enforce the rules of address distribution. We’ll get into the details a bit more below.
 
 First, let’s talk about Urbit ID’s implementation and design goals, then about where we envision it going in the future.
 
-If you’re curious to see exactly how this system works, the code is of course [open source](). Including the source for our [phonemic naming system]() and the [visual representations](). There’s even a [Figma plugin]() for using them. There’s also an interface for interacting with Urbit ID whose source is [open too]().
+If you’re curious to see exactly how this system works, the code is of course [open source](https://github.com/urbit/urbit). Including the source for our [phonemic naming system]() and the [visual representations](https://github.com/urbit/sigil-js). There’s even a [Figma plugin](https://github.com/urbit/sigil-figma-plugin) for using them. There’s also an interface for interacting with Urbit ID whose source is [open too](https://github.com/urbit/bridge).
```

Comment:
The link for the source for the "phonemic naming system" needs to be inserted here, I have no idea what this is referring to.

Reply:
Could be [urbit-ob](https://github.com/urbit/urbit-ob/blob/master/src/internal/co.js). Or the appropriate section from hoon.hoon.

Reply:
I'll use what you've linked!

Reply:
TYVM